### PR TITLE
Detectionlists get user

### PR DESCRIPTION
**Pull request by kiran-chaudhary**
## Status
- [x]  Ready : Add changelog and readme. 
- [x] Ready: Commands `departing employee-get` and `highriskemployee-get`

### CODE REVIEW

File: Packs/Code42/Integrations/Code42/Code42.py [hunk 1]
```diff
@@ -738,6 +746,36 @@ def departingemployee_get_all_command(client, args):
     )
 
 
+@logger
+def departingemployee_get_command(client, args):
+    username = args["username"]
```
Comment (antazoey) [inline Packs/Code42/Integrations/Code42/Code42.py]: They want us to get `get()` whenever possible

File: Packs/Code42/Integrations/Code42/Code42.py [hunk 2]
```diff
@@ -333,6 +333,14 @@ def _get_org_id(self, org_name):
             return org_uid
         raise Code42OrgNotFoundError(org_name)
 
+    def get_departing_employee(self, username):
+        user_id = self._get_user_id(username)
+        return self._get_sdk().detectionlists.departing_employee.get(user_id)
+
+    def get_high_risk_employee(self, username):
+        user_id = self._get_user_id(username)
+        return self._get_sdk().detectionlists.high_risk_employee.get(user_id)
```
Comment (antazoey) [inline Packs/Code42/Integrations/Code42/Code42.py]: You probably want to just return `json` here like the other commands.

Comment (antazoey) [reply Packs/Code42/Integrations/Code42/Code42.py]: See Alan's draft PR for an example

File: Packs/Code42/Integrations/Code42/integration-Code42.yml [hunk 3]
```diff
@@ -0,0 +1,1904 @@
+category: Endpoint
```
Comment (antazoey) [inline Packs/Code42/Integrations/Code42/integration-Code42.yml]: We don't need to commit this file

File: Packs/Code42/Integrations/Code42/Code42_test.py [hunk 4]
```diff
@@ -1380,14 +1410,34 @@ def test_departingemployee_get_all_command_when_no_employees(
...
 
 
+def test_departingemployee_get_command(code42_departing_employee_get_mock):
+    client = create_client(code42_departing_employee_get_mock)
+    cmd_res = departingemployee_get_command(
+        client, {"username": _TEST_USERNAME}
+    )
+    get_func = code42_departing_employee_get_mock.detectionlists.departing_employee.get
+    get_func.assert_called_once_with(_TEST_USER_ID)
+    assert cmd_res.raw_response == _TEST_USERNAME
```
Comment (antazoey) [inline Packs/Code42/Integrations/Code42/Code42_test.py:1585]: Make sure to test the outputs as well as the raw response. You don't have to test human readable outputs though.

Comment (kiran-chaudhary) [reply Packs/Code42/Integrations/Code42/Code42_test.py:1585]: Done.

File: Packs/Code42/Integrations/Code42/Code42.py (hunk 2 above)
Comment (kiran-chaudhary) [reply Packs/Code42/Integrations/Code42/Code42.py]: Changed.

File: Packs/Code42/Integrations/Code42/integration-Code42.yml (hunk 3 above)
Comment (kiran-chaudhary) [reply Packs/Code42/Integrations/Code42/integration-Code42.yml]: Shall I remove this file from the commit?

File: Packs/Code42/Integrations/Code42/README.md [hunk 5]
```diff
@@ -314,6 +314,54 @@ Get all employees on the Departing Employee List.
...
+
+
+#### Command Example
+```!code42-departingemployee-get username="partner.demisto@example.com```
+
+#### Context Example
+```
+{
+    "Code42": {
+        "DepartingEmployee": {
```
Comment (kiran-chaudhary) [inline Packs/Code42/Integrations/Code42/README.md]: To confirm, as I wasn't able to test this. Response doesn't contain the keys `code42` & `DepartingEmployee` but I assume it must be getting added while making response adhere to Demisto format. Hence documented alike other commands.

Comment (alanag13) [reply Packs/Code42/Integrations/Code42/README.md]: Once you get your environment working, you should be able to automatically create the docs using `demisto-sdk generate-docs`.

**Comment by alanag13** [comment]
I tried to test this out, but the command definitions still need to be added to the `Code42.yml` file. On the surface the code looks good but I'd like to verify it properly before approving / merging.

**Comment by kiran-chaudhary** [comment]
> I tried to test this out, but the command definitions still need to be added to the `Code42.yml` file. On the surface the code looks good but I'd like to verify it properly before approving / merging.

Added `Code42.yml` changes and tested too.

**Comment by alanag13** [comment]
@unparalleled-js I have tested this and it works, can you give everything a once-over and approve before we merge?

**Review by antazoey — APPROVED** [review]

**Pull request merged**